### PR TITLE
Add initial support for mv-list/mv-list-item, and rewrite mv-multiple to it

### LINKED ISSUE #72: Collections should be represented in the DOM
Attribute TBD, most likely `data-list`

Reasons:
- `data-multiple` can always be emulated by wrapping the element with a wrapper div (if one does not exist) and putting `data-list` on that div. Mavo can do that automatically.
- Drag and Drop becomes immensely easier to implement when there is a container
- No need for a dummy marker element any more.
- Makes it possible to mark up collections where the repeated part is multiple elements (e.g. definition lists)

### PR DESCRIPTION
**Pull request by LeaVerou**
Closes #72.

Several regressions still, so this is a draft PR for now.

### CODE REVIEW

**Review by DmitrySharabin — APPROVED** [review]
Everything seems quite logical to me.

What are you planning to do with the `mv-cn` and `mv-c` elements? Add the `display: contents` declaration to the stylesheet?

File: src/mavo.js [hunk 1]
```diff
@@ -1000,13 +992,93 @@ if (_.polyfills.length > 0) {
...
 
+// Transition legacy mv-multiple syntax to new mv-list/mv-list-item syntax
+$$("[mv-multiple]").forEach(item => {
+	item.setAttribute("mv-list-item", item.getAttribute("mv-multiple"));
+	console.warn("mv-multiple is deprecated. Please use mv-list-item instead");
+});
+
+// Expand mv-list="foo" to mv-list property="foo" and same for items
+$$("[mv-list]:not([property])").forEach(e => e.setAttribute("property", e.getAttribute("mv-list")));
+$$("[mv-list-item]:not([property])").forEach(e => e.setAttribute("property", e.getAttribute("mv-list-item")));
```
[the remark this answers is not in the document]
Comment (LeaVerou) [reply src/mavo.js:1014]: Yes

**Comment by LeaVerou** [comment]
> What are you planning to do with the `mv-cn` and `mv-c` elements? Add the `display: contents` declaration to the stylesheet?

Oops, that was debugging that I accidentally committed. I was wondering if I should use a custom element or a `<div>`.

File: src/mavo.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (DmitrySharabin) [reply src/mavo.js:1014]: Do you mind correcting the comment then?

`// Expand mv-list="foo" to mv-list="foo" property="foo" and same for items`

**Pull request merged**